project/plugins.sbt

```diff
@@ -8,10 +8,10 @@ addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.6.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
 
-addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.8.2")
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "2.0.4")
 
 addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
 
 addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.29")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.7")
\ No newline at end of file
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.15")
```

Comment:
just wondering if com.typesafe.play plugging upgrade is something required as a part of scala upgrade?